docs/webpack.config.js

```diff
@@ -0,0 +1,60 @@
+var path = require("path");
+var webpack = require("webpack");
+var fableUtils = require("fable-utils");
+
+function resolve(filePath) {
+    return path.join(__dirname, filePath)
+}
+
+var babelOptions = fableUtils.resolveBabelOptions({
+    presets: [["es2015", { "modules": false }]],
+    plugins: ["transform-runtime"]
+});
+
+var isProduction = process.argv.indexOf("-p") >= 0;
+console.log("Bundling for " + (isProduction ? "production" : "development") + "...");
+
+module.exports = {
+    devtool: "source-map",
+    entry: resolve('./docs.fsproj'),
+    output: {
+        filename: 'public/dist/js/bundle.js'
```

Comment:
I'd also use `resolve('public/dist/js/bundle.js')` in case you want to run the command from a different directory.